zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/CamundaExporterIT.java

```diff
@@ -224,6 +229,95 @@ void shouldPeriodicallyFlushBasedOnConfiguration(
     verify(spiedController, times(2)).scheduleCancellableTask(eq(Duration.ofSeconds(2)), any());
   }
 
+  @TestTemplate
+  void shouldExportRoleRecord(
```

Comment:
🔧 There's a new IT class called `CamundaExporterHandlerIT`. I think it would be better and easier to implement this test there.